pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -96,6 +97,26 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 			if warmPool != nil {
 				warmPoolTask.MinSize = warmPool.MinSize
 				warmPoolTask.MaxSize = warmPool.MaxSize
+
+				if warmPool.EnableLifecyleHook {
+					name := ig.Name + "-WarmPoolLifecycleHook"
```

Comment:
I think the convention is to use lower case names and not sure "LifecycleHook" adds any value.
```suggestion
					name := ig.Name + "-warmpool"
```

Reply:
I don't disagree. I went with this because NTH hook had the same convention.

Personally I would prefer just "warmpool" or "kops-warmpool" for example. The name only need to be unique within the ASG, and adding the IG name to it feels excessive

Reply:
I also agree, that we could use just `warmpool`